src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java

```diff
@@ -128,6 +128,11 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
             String message = result.getMsg();
             GHCommitState state = result.getState();
 
+            listener.getLogger().printf(
+                    "[GitHub Commit Status Setter] Going to set status %s on repos %s (sha:%7.7s) with context:%s%n",
```

Comment:
normally prefix should be short and name plugin and then the same UI name i.e. `[GitHub] [getDescriptor().getDisplayName()]`